src/map/clif.cpp

```diff
@@ -25201,6 +25212,19 @@ void clif_set_npc_window_pos_percent(map_session_data& sd, int x, int y)
 #endif  // PACKETVER_MAIN_NUM >= 20220504
 }
 
+/// Displays special popups (ZC_SPECIALPOPUP).
+/// Wokrs only if player moved map to another map.
+/// 0bbe <popup id>.L
+void clif_specialpopup(map_session_data* sd, int id ){
+#if PACKETVER >= 20221005
+	struct PACKET_ZC_SPECIALPOPUP p = { 0 };
+
+	p.PacketType = HEADER_ZC_SPECIALPOPUP;
+	p.ppId = id;
+	clif_send( &p, sizeof( p ), &sd->bl, AREA);
```

Comment:
```suggestion
	clif_send( &p, sizeof( p ), &sd->bl, SELF);
```
It should only be sent to the player itself and not to the area.